packages/cojson/src/coValueCore.ts

```diff
@@ -121,14 +121,35 @@ export class CoValueCore {
     this.header = header;
     this._sessionLogs = internalInitSessions;
     this.node = node;
+  }
+
+  groupInvalidationSubscription?: () => void;
+
+  subscribeToGroupInvalidation() {
```

Comment:
Deferring the group subscription to the first `getCurrentContent` call.

This should both:
- remove the dependency requirement when loading a new CoValue in handleNewContent
- totally skip the group content loading on the sync server giving us a small perf boost